teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(name, age, specialization)
+    super(name: name, age: age)
+    @specialization = specialization
+  end
+
+  def can_use_library?
```

Comment:
- [ ] The name of this method should be `can_use_services?`

Reply:
@DammyShittu, please I need your assistance as I am getting into my last chance

Reply:
@TedLivist, please I need your assistance

Reply:
Hi @yayaismaail. You have reverted to the previous changes, now all you need to do as requested in the second change, in the Person class, simply change line 13 from this

```
of_age? || parent_permission
```
to this
```
of_age? || @parent_permission
```

Reply:
@DammyShittu, thanks a lot